platform/android/app/src/main/assets/legal

```diff
@@ -0,0 +1 @@
+../../../../../../assets/legal
```

Comment:
???

Reply:
Changed the symbolic link, but GitHub is showing it funny I guess